Add scraped page archiver

ondenman opened the pull request:
## What does this do?

Uses scraped-page-archive to archive all pages scraped.

## Why is this needed?

There's no specific urgency on this as there isn't an election until 2021, but the goal is to add this to all scrapers eventually.

## Relevant Issue(s):

n/a

## [Scraper Change checklist](https://github.com/everypolitician/everypolitician/wiki/Scraper-Change-checklist)
* [ ] 1. scraper is on Morph.io under the "everypolitician-scrapers" group? — no, it's still at https://morph.io/tmtmtmtm/welsh-assembly, but we can move it separately later
* [x] 2. scraper's GitHub "Website" link points at morph.io page?
* [x] 3. scraper is set to auto-run?
* [x] 4. scraper is archiving? — that's what this is doing!
* [x] 5. legislature has a scraper webhook set? -- on https://morph.io/tmtmtmtm/wales-AMs-wikidata

## [Adding Archiving to Scraper checklist](https://github.com/everypolitician/everypolitician/wiki/Adding-Archiving-to-Scraper-checklist)
* [x] 1. we are using at least version 0.5 of scraped-page-archive-gem?
* [x] 2. scraper uses scraped-page-archive gem directly or via a suitable strategy? — directly
* [ ] 3. MORPH_SCRAPER_CACHE_GITHUB_REPO_URL is configured? — @tmtmtmtm The scraper's ENV variables need configuring on Morph
* [x] 4. pages are being archived in new branch of correct scraper repo?

## [Gemfile checklist](https://github.com/everypolitician/everypolitician/wiki/Gemfile-checklist/)
* [x] 1. all links are secure
* [x] 2. links to Github use `github:` protocol, not simply `git:`
* [x] 3. formatting is consistent with our normal Rubocop setup

ondenman committed:
Add scraped_page_archive gem

ondenman committed:
Bundle install

ondenman committed:
Use scraped_page_archive gem

tmtmtmtm commented:
There aren't elections due until 2021(!), so there's no immediate rush on this as part of the current task, but it's worthwhile adding this for all scrapers of official sites. It would be good to update the description here to the same format as the other scrapers (and in particular to include the one for updating a Gemfile), which would then show what else still needs to be done here.

ondenman committed:
Tidy Gemfile for Rubocop

ondenman committed:
Add secure git gem source

ondenman committed:
Fix Github gem source

Source was pointing to :git not :github source as defined above

(Github.lock is unchanged as the correct URL is set.)

tmtmtmtm commented:
👍 

Scraper now configured and running.
